### PR TITLE
@broskoski: Artwork updates

### PR DESCRIPTION
**Pull request by dzucconi**
More of the same. You've got a Force PR incoming.

### CODE REVIEW

File: lib/helpers.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import {
+  assign,
+} from 'lodash';
+
+export function enhance(xs = [], source = {}) {
+  return xs.map(x => assign(source, x));
+}
```
Comment (dzucconi) [inline lib/helpers.js]: I keep defining little utility functions like https://github.com/artsy/metaphysics/blob/8f8ff3709b4ae6544dc1eb962d25bbf853bb4e88/schema/image/normalize.js#L16. This is gonna be my utility belt?

Comment (broskoski) [inline lib/helpers.js]: lol @ enhance

File: schema/artwork/index.js [hunk 2]
```diff
@@ -180,9 +204,24 @@ const ArtworkType = new GraphQLObjectType({
...
-            fairs.map(fair => fair.related_type = 'Fair'); // eslint-disable-line no-param-reassign
-            sales.map(sale => sale.related_type = 'Sale'); // eslint-disable-line no-param-reassign
-            return _.first(_.take(fairs.concat(sales)));
+            const relatedFairs = enhance(fairs, { related_type: 'Fair' });
+            const relatedSales = enhance(sales, { related_type: 'Sale' });
+            return _.first(_.take(relatedFairs.concat(relatedSales)));
+          });
+        },
+      },
+      highlights: {
```
Comment (broskoski) [inline schema/artwork/index.js]: :white_check_mark:

File: schema/fields/markdown.js [hunk 3]
```diff
@@ -20,8 +24,9 @@ export default {
         sanitize: false,
         smartypants: false,
       }));
-      return marked(obj[fieldName] || '');
+      return marked(value);
```
Comment (broskoski) [inline schema/fields/markdown.js]: Yes

**Pull request merged**